### PR TITLE
Capstone project 1-Completion

### PR DESCRIPTION
**Pull request by oshanedesign**
Completed milestone requirements of mobile, desktop, popup menu and dynamic javascript for this project. 

Please see recorded video link: https://www.loom.com/share/316f80cb9b024dceb1ba01a0caba7da5

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+      <img src="images/naver.PNG" alt="naver">
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6  text-right pt-2">
+      <img src="images/daum.PNG" alt="daum">
+    </div>
+    <div class="col-6">
+      <img src="images/airbnb.PNG" alt="airbnb">
+    </div>
```
Comment (keroloussamy) [inline about.html:192]: - [ ] Kindly your images are overflows the page, kindly control the width of every image to not overflow and break your design.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @oshanedesign, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+  display: none;
+}
+
+.desktop-partner-section {
+  display: none;
+}
+
+.footer {
+  display: none;
+}
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply style.css:140]: don
footer desktop
e

File: index.html [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+          <img class="desktop-speaker-img" src="images/speaker_06.png" alt="">
+        </div>
+        <div class="col-4">
+          <p class="speaker-name">SohYeong Noh</p>
+              <p class="speaker-profession">Director of Art Centre Nabi and a board member of CC Korea</p>
+              <span><i class="fa-solid fa-ellipsis"></i></span>
+              <p class="speaker-experience">As the author of Digital Art  of Our Time, he opened 'Art Center Nabi', Korea's first digital art institution in 2000, and is currently serving.</p>
+        </div>
+      </div>
+     </section> -->
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply index.html:368]: done

[the remark this answers is not in the document]
Comment (oshanedesign) [reply index.html:368]: done 
width

File: index.html [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+        <div class="row">
+          <div class="col text-secondary text-center pt-5">Partner</div>
+          </div>
+          <div class="row">
+          <div class="col text-center line pb-5"><i class="fa-solid fa-ellipsis"></i><i class="fa-solid fa-ellipsis"></i></div>
+        </div>
+        <div class="row text-center pb-5">
+          <div class="col">
+            <img src="images/partner_logo.PNG" alt="">
+          </div>
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply index.html:387]: done
width

File: index.html [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+<div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+  <ol class="carousel-indicators">
+    <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+  </ol>
+  <div class="carousel-inner transparent">
+    <div class="carousel-item active text-center">
+      <img src="images/title.png" class="d-block w-100 slides" alt="...">
+    </div>
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply index.html:87]: done

File: style.css [hunk 6]
```diff
@@ -0,0 +1,543 @@
...
+
+  .desktop-event-2013,
+  .desktop-event-2011 {
+    width: 100%;
+    height: auto;
+  }
+
+  .footer {
+    display: none;
+  }
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply style.css:528]: done

File: about.html [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+  <header class="header">
+    <div class="container-fluid bg-white">
+     <div class="row my-4">
+        <div class="col-5 mobile-nav"> 
+          <div id="navToggle" class="mobile-menu-parent"><img class="mobile-menu navIcon" src="images/mobile_menu.png" alt="mobile-menu"></div>
+          <div class="mobile-menu-parent"><i class="fa-solid fa-xmark navIcon hidden"></i></div>
+        </div>
+        <div class="col-7">
+          <a href="#" class="logo-parent"><img class="logo" src="images/logo.png" alt="CC Global Summit 2015 logo"> </a>
+        </div>
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply about.html:41]: done

File: about.html [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+      </ul> 
+    </nav>
+  </div>
+    </div>
+  </header>
+<!-- Main about information Section Start-->
+<section class="main-invite"> 
+  <div class="container-fluid">
+    <p class="hello-text-about">"Hello! Sharing World!"</p>
+    <img class="title-about" src="images/about_title.png" alt="title">
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply about.html:81]: done 
screen

File: about.html [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+      <div class="col">
+        <p class="mobile-about-logo-paragraph">The logo of CC Global Summit 2015 was decided through the logo competition from June 8 to July 7.</p>
+        <p class="desktop-about-logo-paragraph">The logo of CC Global Summit 2015 was decided through the logo competition from June 8 to July 7. After consultation with the winning designer, the logo of CC Global Summit 2015, which was optimized for use on the website and promotional materials, is as follows.</p>
+        
+      </div>
+    </div>
+    <div class="row align-items-center mx-5 mb-5 mt-5 ">
+      <div class="col">
+        <img class="logo-about" src="images/about_logo.png" alt="about-logo">
+      </div>
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply about.html:121]: done 
center

File: about.html (hunk 1 above)
Comment (oshanedesign) [reply about.html:192]: done
de

**Review by Lordkaito — APPROVED** [review]
Hi @oshanedesign, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**